adviser/base/openshift-templates/adviser.yaml

```diff
@@ -188,6 +227,20 @@ objects:
                     - name: "THOTH_DEPLOYMENT_NAME"
                       value: "{{workflow.parameters.deployment_name}}"
 
+              - name: "unresolved-package-handler"
+                dependencies:
+                  - "advise"
+                templateRef:
+                  name: "unresolved-package-handler"
+                  template: "unresolved-package-handler"
+                arguments:
+                  artifacts:
+                    - name: outputdocument
+                      from: "{{tasks.advise.outputs.artifacts.outputdocument}}"
+                  parameters:
+                    - name: THOTH_DOCUMENT_ID
+                      value: "{{workflow.parameters.THOTH_DOCUMENT_ID}}"
+
```

Comment:
sorry let's remove those, as templates are missing in thoth-application

Reply:
thanks for pointing these @pacospace 
will update this in new pr.